packages/fiori/cypress/specs/FCL.cy.tsx

```diff
@@ -104,9 +106,6 @@ describe("Columns resize", () => {
 			.find(".ui5-fcl-column--end")
 			.should("have.class", "ui5-fcl-column--hidden");
 
-		cy.wrap({ setAnimationMode })
```

Comment:
It looks to me that this test check changes of animation mode dynamically. If that isn't possible we may as well delete it. 
The  setAnimationMode call serves as an Act phase and triggers the rerendering, perhaps we need to think of another way to trigger it, like changing the layout.

Reply:
Nevertheless, I find this test strange. It seems not to rerender the component here, perhaps it assumed that setting new animation mode will do. Maybe we can change the layout to TwoColumnsMidExpanded here. This will trigger rerendering and assertions below will make sense.

```ts
// act: trigger a rerender
cy.get("@fcl")
	.invoke("prop", "layout", "TwoColumnsMidExpanded");
```